SPECS/sm.spec

```diff
@@ -417,12 +417,10 @@ cp -r htmlcov %{buildroot}/htmlcov
 %{_unitdir}/linstor-monitor.service
 
 %changelog
-* Tue Jun 22 2021 Samuel Verschelde <stormi-xcp@ylix.fr> - 2.30.4-1.1
+* Tue Jun 22 2021 Ronan Abhamon <ronan.abhamon@vates.fr> - 2.30.4-1.1
 - Sync with hotfix XS82E028
-- Update XCP-ng patches
-- *** Upstream changelog ***
-- * Wed May 19 2021 Mark Syms <mark.syms@citrix.com> - 2.30.4-1
-- - CA-354228: Reinstate load calls in _pathrefresh
+- CA-354228: Reinstate load calls in _pathrefresh
```

Comment:
Please keep the ustream changelog as I formatted it. This is the new way we use to embed upstream changelogs since a few other hotfixes to other RPMs.